Roychela/Politicalapp
# #163674338 Made changes to all UI pages

**What does this PR do?**
Makes changes to the UI pages.

**Describe the tasks to be completed**.

- [x] Make changes to each UI page.

- [x] Add an images folder with images.

**How can this be manually tested?**
Clone the repo. Open the UI directory and view pages in the browser to see changes.

**What are the relevant Pivotal Tracker stories?**
https://www.pivotaltracker.com/story/show/163674338
